commercetools-models/src/main/java/io/sphere/sdk/apiclient/expansion/ApiClientExpansionModelImpl.java

```diff
@@ -0,0 +1,8 @@
+package io.sphere.sdk.apiclient.expansion;
+
+import io.sphere.sdk.expansion.ExpansionModelImpl;
+
+final class ApiClientExpansionModelImpl<T>  extends ExpansionModelImpl<T> implements ApiClientExpansionModel<T> {
```

Comment:
Not needed!?

Reply:
since i cant get rid of `ApiClientExpansionModel` i need an implementation class for it.